### ISSUE TITLE
High Elf Starting Weapon Inconsistency

### ISSUE THREAD

**Issue by JoLiKMC** [slowtorta/turtlewow-bug-tracker#1535]
Greetings.  I'm here to report a bug that seems to have slipped under the radar.

When creating a High Elf character, the characters are equipped with one kind of weapon, but when the character is finalized, they end up with something else.  Which this is largely a "cosmetic" thing, there are a couple of cases in the weapons aren't even equivalent to each other.  One weapon model, in particular, also doesn't appear to be in the game anywhere else.

Here are the weapons that appear in the Character Creator, followed by what the player actually ends up with:

Warrior
**Scratched Claymore** -> Worn Shortsword, Worn Thalassian Shield

Hunter
**Scuffed Dagger** -> Worn Dagger

Rogue
**Scuffed Dagger** -> Worn Dagger

Priest
**Blackrock Mace** (?) -> Worn Mace
(The weapon in question _looks_ like a Blackrock Mace.  No other model in Vanilla matches it.)

Mage
**Apprentice's Staff** (?) -> Bent Staff
(The weapon in question _looks_ like the Apprentice's Staff from _The Burning Crusade._  No other model in Vanilla matches it.)

Paladin
**Scratched Claymore** -> Battleworn Bastard Sword

**Steps to reproduce the behavior with as much detail as possible:**

1. Log in to _Turtle WoW_ using the official game client.
2. Click "Create New Character".
3. Select "High Elf" for the race.  Gender, Class, and Name don't matter.
4. Take note of the weapon in the character's hand, then click "Accept".
5. Examine the newly-created character.  The weapon they start with is different from what was shown in Step 4.

**Expected behavior. Describe how it should work:**
Simply put, unless _Turtle WoW_ is implementing some sort of "This is how your character _could_ look" system like Retail has, the characters should begin life with the weapons they were displayed with in the Character Creation screen.  And that's it.

**Comment by slowtorta** [comment]
This will be fixed in the upcoming patch 1.16.1!

**Comment by JoLiKMC** [comment]
**This has not been fixed as of v1.16.1.**

The "character creation preview" weapons and the actual weapons a high elf gets upon finalization are still _entirely different_.

**Relevant image:**
![116-1_HelfWeaponry](https://user-images.githubusercontent.com/102710555/180888227-acd05618-9584-43b4-9c4b-e9522f6efc37.jpg)
